### PR TITLE
ports create and destroy

### PR DESCRIPTION
**Pull request by harryob**
this unit test is designed to increase CI times arbitrarily by 15 minutes

actually it picks up hard dels, bad initialisations and bad deletes

this pr will have morbillions of CI errors but that's intentional until people (me) fix all the beautiful harddels

you (the reader) can do this yourself, check out the guide https://github.com/tgstation/tgstation/blob/master/.github/guides/HARDDELETES.md

:cl:
fix: fixed a bunch of bad dels causing lag
/:cl:

### CODE REVIEW

**Review by fira — APPROVED** [review]

File: code/datums/paygrades/factions/uscm/marine.dm [hunk 1]
```diff
@@ -93,11 +93,11 @@
...
 
-/datum/paygrade/marine/e9s
-	paygrade = "ME9S"
-	name = "Sergeant Major of the Colonial Marine Corps"
+/datum/paygrade/marine/e9c
+	paygrade = "ME9C"
+	name = "Sergeant Major"
 	prefix = "SMCMC"
-	rank_pin = /obj/item/clothing/accessory/ranks/marine/e10c
+	rank_pin = /obj/item/clothing/accessory/ranks/marine/e9c
```
Comment (Watermelon914) [inline code/datums/paygrades/factions/uscm/marine.dm:100]: Seems out of scope for what this PR is doing.

**Review by Watermelon914 — CHANGES_REQUESTED** [review]
